pages/job-category/index.js

```diff
@@ -89,7 +91,7 @@ export default function createJobcategory() {
       <div class="card">
```

Comment:
```suggestion
      <div>
```
Please don't mix pure CSS with styled-components. 🙏 

By the way, everything that follows the same pattern as line 89 to 125 could be the same component given different props (if just the source of the image changes).